applications/DG-Max/Utils/Symmetry/Domain.h

```diff
@@ -0,0 +1,132 @@
+/*
```

Comment:
How does this fit in?

Reply:
Crap, that should not have been committed.